components/ui/Socialmedia.js

```diff
@@ -0,0 +1,64 @@
+import styles from "@/components/ui/Footer.module.css";
+import React from "react";
+
+const SocialLinks = ({ url, title, icon }) => {
```

Comment:
```suggestion
const SocialLink = ({ url, title, icon }) => {
```
Keep the plural names for components containing multiple items.